src/python/examples/image_client.py

```diff
@@ -345,6 +345,11 @@ def convert_http_metadata_config(_metadata, _config):
     max_batch_size, input_name, output_name, c, h, w, format, dtype = parse_model(
         model_metadata, model_config)
 
+    supports_batching = max_batch_size > 0
+    if not supports_batching and FLAGS.batch_size != 1:
+        print("ERROR: This model doesn't support batching.")
+        sys.exit(1)
```

Comment:
equivalent of C++ client check here: https://github.com/triton-inference-server/client/blob/cb7c86dbe5e8cf70ec51d70a706d0958b1b09de9/src/c%2B%2B/examples/image_client.cc#L450-L454